app/app.go

```diff
@@ -1138,6 +1170,7 @@ func initParamsKeeper(
 	paramsKeeper.Subspace(ibctransfertypes.ModuleName)
 	paramsKeeper.Subspace(ibcexported.ModuleName)
 	paramsKeeper.Subspace(icahosttypes.SubModuleName)
+	paramsKeeper.Subspace(packetforwardtypes.ModuleName).WithKeyTable(packetforwardtypes.ParamKeyTable())
```

Comment:
This is not required in v7.1.1 (x/params is migrated out due to SDK v0.51 deprecation).

It is fine being here, just not used

Reply:
Thanks for the comment @Reecepbcups . 
Paramspace is still working in v0.47 and v0.50. I'm not sure if the IBC requires it or not.

Reply:
I've checked Osmosis, which migrated to v0.47 recently, and they also setup PFM

Reply:
I mean, their Osmosis PFM integration also uses params keeper.